app/javascript/dashboard/routes/dashboard/settings/personal/Index.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <div class="flex items-center w-full overflow-y-auto">
+    <div class="flex flex-col h-full p-5 pt-16 mx-auto my-0 font-inter">
+      <div class="flex flex-col gap-16 sm:max-w-[720px]">
+        <div class="flex flex-col gap-6">
+          <h2 class="mt-4 text-2xl font-medium text-ash-900">
+            {{ $t('PROFILE_SETTINGS.TITLE') }}
+          </h2>
+          <user-profile-picture
+            :src="avatarUrl"
+            :name="name"
+            size="72px"
+            @change="updateProfilePicture"
+            @delete="deleteProfilePicture"
+          />
+          <user-basic-details
+            :name="name"
+            :display-name="displayName"
+            :email="email"
+            :email-enabled="!globalConfig.disableUserProfileUpdate"
+            @update-user="updateProfile"
+          />
+        </div>
+
+        <base-personal-item
+          :header="$t('PROFILE_SETTINGS.FORM.MESSAGE_SIGNATURE_SECTION.TITLE')"
+          :description="
+            $t('PROFILE_SETTINGS.FORM.MESSAGE_SIGNATURE_SECTION.NOTE')
+          "
+        >
+          <template #settingsItem>
+            <message-signature
+              :message-signature="messageSignature"
+              @update-signature="updateProfile"
+            />
+          </template>
+        </base-personal-item>
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+import globalConfigMixin from 'shared/mixins/globalConfigMixin';
+import uiSettingsMixin, {
+  isEditorHotKeyEnabled,
+} from 'dashboard/mixins/uiSettings';
+import alertMixin from 'shared/mixins/alertMixin';
+import { mapGetters } from 'vuex';
+import { clearCookiesOnLogout } from 'dashboard/store/utils/api.js';
+
+import UserProfilePicture from './UserProfilePicture.vue';
+import UserBasicDetails from './UserBasicDetails.vue';
+import MessageSignature from './MessageSignature.vue';
+import BasePersonalItem from './BasePersonalItem.vue';
+
+export default {
+  components: {
+    MessageSignature,
+    BasePersonalItem,
+    UserProfilePicture,
+    UserBasicDetails,
+  },
+  mixins: [alertMixin, globalConfigMixin, uiSettingsMixin],
+  data() {
+    return {
+      avatarFile: '',
+      avatarUrl: '',
+      name: '',
+      displayName: '',
+      email: '',
+      alertMessage: '',
+      messageSignature: '',
+    };
+  },
+  computed: {
+    ...mapGetters({
+      currentUser: 'getCurrentUser',
+      currentUserId: 'getCurrentUserID',
+      globalConfig: 'globalConfig/get',
+    }),
+  },
+  watch: {
+    currentUserId(newCurrentUserId, prevCurrentUserId) {
+      if (prevCurrentUserId !== newCurrentUserId) {
+        this.initializeUser();
+      }
+    },
```

Comment:
Why do we need this? Since a user would be able to access their profile only, so there shouldn't be a reason that the `userId` changes